rosbag2_storage/test/rosbag2_storage/plugin_constants.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
maybe a detail, but consider renaming this file to `test_plugin_constants.hpp` to stay in sync with `test_plugin.hpp` and to highlight that it's clearly used only in the test plugin.